src/main/java/com/e_commerce/backend/models/Product.java

```diff
@@ -1,13 +1,15 @@
 package com.e_commerce.backend.models;
 
 import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
 import jakarta.persistence.*;
 import lombok.Data;
 import lombok.RequiredArgsConstructor;
 import org.hibernate.annotations.CreationTimestamp;
 import org.hibernate.annotations.UpdateTimestamp;
 
 import java.time.LocalDateTime;
+import java.util.List;
 
 @Entity
 @Table(name = "products")
```

Comment:
Product is something that you call everytime to make your calls faster you should add some form of indexing (Research indexing)